SMLHelper/Patchers/OptionsPanelPatcher.cs

```diff
@@ -94,5 +110,167 @@ internal static void AddTabs_Postfix(uGUI_OptionsPanel __instance)
                 }
             }
         }
+
+
+
+        // do not show tab if it is already visible (to prevent scroll position resetting)
+        internal static bool SetVisibleTab_Prefix(uGUI_TabbedControlsPanel __instance, int tabIndex)
+        {
+            return !(tabIndex >= 0 && tabIndex < __instance.tabs.Count && __instance.tabs[tabIndex].pane.activeSelf);
+        }
+
+        // adjusting ui elements
+        internal static void SetVisibleTab_Postfix(uGUI_TabbedControlsPanel __instance, int tabIndex)
+        {
+            if (tabIndex >= 0 && tabIndex < __instance.tabs.Count)
+            {
+                try
+                {
+                    Transform options = __instance.tabs[tabIndex].container.transform;
+
+                    for (int i = 0; i < options.childCount; ++i)
+                    {
+                        Transform option = options.GetChild(i);
+
+                        if (option.localPosition.x == 0) // ui layout didn't touch this element yet
+                            continue;
+
+                        if (option.name.Contains("uGUI_ToggleOption"))
+                            ProcessToggleOption(option);
+                        else
+                        if (option.name.Contains("uGUI_SliderOption"))
+                            ProcessSliderOption(option);
+                        else
+                        if (option.name.Contains("uGUI_ChoiceOption"))
+                            ProcessChoiceOption(option);
+                        else
+                        if (option.name.Contains("uGUI_BindingOption"))
+                            ProcessBindingOption(option);
+                    }
+                }
+                catch (System.Exception e)
+                {
+                    V2.Logger.Log($"Exception while adjusting mod options: {e.GetType()}\t{e.Message}", LogLevel.Error);
+                }
+            }
+        }
+
+        static void ProcessToggleOption(Transform option)
+        {
+            Transform check = option.Find("Toggle/Background");
+            Text text = option.GetComponentInChildren<Text>();
+
+            // :)
+            if (text.text == "Enable AuxUpgradeConsole                        (Restart game)")
+                text.text = "Enable AuxUpgradeConsole (Restart game)";
```

Comment:
Remove.
There is no need to explicitly fix the mistakes of other mods.

I will make an update to MoreCyclopsUpgrades to coincide with the release of SMLHelper 2.3.